pkg/sysinfo/cgroup2_linux.go

```diff
@@ -66,6 +69,25 @@ func newV2(quiet bool, opts *opts) *SysInfo {
 	return sysInfo
 }
 
+func getSwapLimitV2() bool {
+	groups, err := cgroups.ParseCgroupFile("/proc/self/cgroup")
+	if err != nil {
+		return false
+	}
+
+	g := groups[""]
+	if g == "" {
+		return false
+	}
+
+	g = path.Clean(g)
+	cGroupPath := filepath.Join("/sys/fs/cgroup", g, "memory.swap.max")
```

Comment:
Given that this is linux-only, this should probably use `path.Join()` instead of `filepath.Join()` (also prevents the extra import being added

Reply:
ok